src/test/java/com/google/api/codegen/testdata/discogapic/java/java_simplecompute.v1.json.baseline

```diff
@@ -5572,34 +5661,75 @@ import javax.annotation.Generated;
  */
 @Generated("by GAPIC v0.0.5")
 public class HttpAddressAdminStub extends AddressAdminStub {
+  private static final String BASE_URL = "https://www.googleapis.com/compute/v1/projects/";
```

Comment:
The "projects/" part is logically part of the resource path, not the base URL

Reply:
That value is pulled straight from the root Discovery document. I don't think it's worth deducing which part of the given base url is actually the base url, since concatenating a built PathTemplate to the default base url works fine.

Reply:
Hmm I strongly disagree with their design but I guess we can't really do much about that.